src/data/translationProgress.json

```diff
@@ -3,637 +3,637 @@
     "languageId": "af",
     "words": {
       "approved": 2328,
-      "total": 340424
+      "total": 340423
```

Comment:
@wackerow is this unusual behaviour?

Reply:
We may have just removed some strings, but not sure the details.